docs/general/events.md

```diff
@@ -17,74 +17,55 @@ network connection severed, etc.
 
 In OpenTelemetry, Events are implemented as a specific type of [`LogRecord`](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.44.0/specification/logs/api.md) that conforms to the conventions included here.
 
-In addition to a required name, an Event may contain a _payload_ (body) of any type permitted
-by the [LogRecord body](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#field-body).
-In its implementation, the Event _payload_ (body) will constitute the `Body` of the `LogRecord`.
-Like all other OpenTelemetry signals, an Event has optional attribute metadata that helps describe
-the event context.
+Events SHOULD be generated / produced / recorded using the
+[Logs API](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.44.0/specification/logs/api.md#emit-a-logrecord)
+to ensure that the event is created using the configured SDK instance.
 
-Over time, some Events will be specified by OpenTelemetry and will have documented payload structure,
-field semantics, and stability and requirement levels. Other events may be user-defined and carry
-bespoke user semantics. When an Event name exists in the semantic conventions, its _payload_
-structure and semantics will also be defined.
+OpenTelemetry Semantic Conventions that define events SHOULD document the event name along
+with attributes and the type of the body if any.
```

Comment:
I think that it should also define severity level (even if this would be usually INFO).